### DISCUSSION TITLE
TRMM / Mesh agent mismatch - macOS

### DISCUSSION THREAD

**Discussion by ghost** [amidaware/tacticalrmm#1447] [General]
110+ TRMM agents installed and the number of Mesh entries is out by 6 machines (all Mac machines). 

Without comparing TRMM to Mesh is there a way to bulk resync the two lists?

I'd be happy to bash script any fix.

**Comment by silversword411** [comment]
Mesh maintains it's own lists. If the mac isn't connecting to meshcentral, it won't appear. As soon as it does, it'll reappear/add itself to the list. 

If you're missing the devices from mesh, use it's debug to figure out why (try restarting meshcentral on TRMM server, might be related to https://github.com/Ylianst/MeshCentral/issues/2831
